homework-calcu-D03-W01.js

```diff
@@ -0,0 +1,11 @@
+function calcu(num1,num2){
+ 
+    var add , sub , divide , multiply ;
+    add = " add = " + (num1 + num2)  +  "\n";
+    sub = "subtract = " + (num1 - num2) + "\n"  ;
+    divide ="divide = " + num1 / num2 +  "\n" ;
+    multiply = "multiply = " + num1 * num2 + "\n" ;
+        return add + sub + divide + multiply;
+    }
+    
+calcu(1,2);
```

Comment:
So the calculator was meant to be just like a regular calculator. Pass in two numbers and an operation and get the result back. For example,
```js
calculator(2, 3, "add");
// returns 5
calculator(2, 3, "subtract");
// returns 1
...
```